plugins/basic/info.yml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+
+# This file contains the plug-in configuration data that will be packaged
+# into manifest.py by the build_plugin.py script.
+
+
+#
+# Plug-in information recommended values to specify by convention.
+#
+
+# Plug-in name that will be passed to 'sgtk.LogManager.get_logger()'
+# and will prefix the plug-in messaqges displayed in Maya script editor.
+name: "tk-maya-plugin-basic"
```

Comment:
This doesn't work with our general naming convention for apps in toolkit. I think we should just call it `tk-maya-basic` for simplicity and name the python module `tk_maya_basic` etc.